web/app/js/components/TapEventTable.jsx

```diff
@@ -42,28 +42,29 @@ let tapColumns = (filterOptions, ResourceLink) => [
   },
   {
     title: "Direction",
+    key: "direction",
     dataIndex: "base.proxyDirection",
+    width: "60px",
     filters: [
       { text: "Inbound", value: "INBOUND" },
       { text: "Outbound", value: "OUTBOUND" }
     ],
+    render: directionColumn,
     onFilter: (value, row) => _.get(row, "base.proxyDirection").includes(value)
   },
   {
-    title: "Source",
-    key: "source",
-    dataIndex: "base",
-    filters: genFilterOptionList(filterOptions.source),
-    onFilter: (value, row) => row.base.source.pod === value || row.base.source.str === value,
-    render: d => srcDstColumn(_.get(d, "source"), _.get(d, "sourceMeta.labels", {}), ResourceLink)
-  },
-  {
-    title: "Destination",
-    key: "destination",
-    dataIndex: "base",
-    filters: genFilterOptionList(filterOptions.destination),
-    onFilter: (value, row) => row.base.destination.pod === value || row.base.destination.str === value,
-    render: d => srcDstColumn(_.get(d, "destination"), _.get(d, "destinationMeta.labels", {}), ResourceLink)
+    title: "Source/Destination",
+    key: "src-dst",
```

Comment:
since the direction column already conveys src/dst info, what do you think about just calling this column `name`, or, `resource`? in the CLI we just call it `NAME`.

Reply:
ooh, yeah, Name could be good!